build.sbt

```diff
@@ -21,8 +21,8 @@ val scalafixV = "0.11.1"
 ThisBuild / crossScalaVersions := Seq(Scala213, "2.12.18")
 ThisBuild / scalaVersion := Scala213 // the default Scala
 ThisBuild / githubWorkflowJavaVersions := Seq(
-  JavaSpec.temurin("8"),
-  JavaSpec.temurin("11")
+  JavaSpec.temurin("11"), // the first java is the default java, don't change the order
```

Comment:
If I'm reading this correctly, it will publish with Java 11.  I _think_ that's fine for internal projects (and would like to know any where it's not, because this keeps coming up, and it's long past time).  But if we proceed with that, I don't see a point to testing on Java 8.

Reply:
I don't _think_ this is true, because `tlJdkRelease` is still java 8, and that's what sets the target jvm version for the compiled code (I didn't know this yesterday, I've just been asking Arman about it 😅)

Reply:
Okay, that should work.  I knew about that setting, but its default value comes as a surprise.  I suggest we set that one explicitly.

I'm going to start identifying projects on 8 and chasing them upward, because there are too many variants of this conversation right now for no known (to me) users.

Reply:
Okay, just pushed that change!